modules/administration-guide/partials/con_external-postgresql-setup.adoc

```diff
@@ -0,0 +1,34 @@
+[id="external-postgresql-setup_{context}"]
+= External PostgreSQL setup
+
+By default, {prod-short} Operator creates and manages deployment of the internal PostgreSQL database. However, {prod-short} Operator does not support full life-cycle capabilities such as backups and recovery.
+
+An external PostgreSQL database is recommended for a business-critical setup. The external PostgreSQL can be configured for High Availability (HA) and Point In Time Recovery (PITR).
+
+The external PostgreSQL instance can be configured as on-premises or to use a cloud service such as Amazon Relational Database Service (Amazon RDS). Amazon RDS, for example, enables deployment of production databases in a Multi-Availability Zone configuration for a resilient disaster recovery strategy with daily and on-demand snapshots.
+
+The recommended configuration of an example database:
```

Comment:
```suggestion
The following example shows a database configuration for business-critical data:
```
Per ISG and SSG: Avoid `recommend` and instead state the user benefit or the purpose.